### PR TITLE
[EC-1086] The Remove individual vault Enterprise Policy prevents Service Users and Provider admins from creating new vault items via the Provider Portal.

### PR DESCRIPTION
**Pull request by coroiu**
## Type of change

```
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

When single vault policy was in effect the `organizationId` that the user belonged to was was always set for new ciphers. This was overwriting the client organization's id when a provider was trying to create new items in their clients vault.

## Code changes

Check that the `AddEditComponent` hasn't already been provided with an organization id before fetching `organizationId` from policy settings. Organization ids are only provided inside of the org-vault and the user is only allowed to access org-vaults of organizations they belong to or are otherwise allowed to administrate, as such the organization id must be a valid id if supplied.

## Screenshots

image

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/web/src/app/vault/org-vault/vault.component.ts [hunk 1]
```diff
@@ -257,6 +257,7 @@ export class VaultComponent implements OnInit, OnDestroy {
 
     const defaultComponentParameters = (comp: AddEditComponent) => {
       comp.organization = this.organization;
+      comp.organizationId = this.organization.id;
```
Comment (eliykat) [inline apps/web/src/app/vault/org-vault/vault.component.ts:260]: This is also set in the `additionalComponentParameters` callback passed in by `cloneCipher` and `addCipher`. If it's now included in the `defaultComponentParameters`, you can remove it from the additional params specified by those callers.

**Review by eliykat — CHANGES_REQUESTED** [review]
These components 🤯

File: apps/web/src/app/vault/org-vault/vault.component.ts (hunk 1 above)
Comment (coroiu) [reply apps/web/src/app/vault/org-vault/vault.component.ts:259]: True, fixed!

**Review by eliykat — APPROVED** [review]
Nice work!

**Review by r-tome — APPROVED** [review]
Looks great!

**Pull request merged**